src/backends/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .backend_interface import BackendInterface
```

Comment:
```suggestion
from .backend_interface import BackendInterface, ServiceType
```

This will let all `from backend import x` work in other files without the relative `.`